traitsui/wx/editor.py

```diff
@@ -61,17 +61,6 @@ def update_editor(self):
         if self.control.GetValue() != new_value:
             self.control.SetValue(new_value)
 
-    def error(self, excp):
-        """Handles an error that occurs while setting the object's trait value."""
-        dlg = wx.MessageDialog(
-            self.control,
-            str(excp),
-            self.description + " value error",
-            wx.OK | wx.ICON_INFORMATION,
-        )
-        dlg.ShowModal()
-        dlg.Destroy()
```

Comment:
pyface also seems to just call `open` on the `MessageDialog` which does https://github.com/enthought/pyface/blob/8adf2a1db26e57cafafd4d2a884d1deff5f35a3d/pyface/i_dialog.py#L149

as opposed to calling wx.MessageDialog methods:
```
dlg.ShowModal()
dlg.Destroy()
```

seems like the right thing to do though.

Reply:
I think Pyface should be doing the equivalent of that..